pitch.md

```diff
@@ -46,6 +194,28 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I've already mentioned some of the other tools/DSLs hanging out in this domain.
+A brief list:
+   - [tkz-euclide](https://www.ctan.org/pkg/tkz-euclide?lang=en) is a LaTeX
+     package which allows for geometrically precise drawings. It is somewhat
+     cryptic and documented only in French :heart: .
```

Comment:
Following this great tradition, I think your DSL's documentation should be entirely in German.